test/utils.py

```diff
@@ -584,6 +584,14 @@ def is_cluster_app_supported():
     return major >= 3
 
 
+def skip_if_tcm_not_supported():
+    if not is_tarantool_ee():
+        pytest.skip("Tarantool Enterprise required")
+
+    if is_tarantool_less_3():
+        pytest.skip("TCM not binary")
```

Comment:
Message don't match with check. May be need something like this?
```suggestion
        pytest.skip("TCM not supported")
```

Reply:
done